server/manager/src/main/java/org/apache/accumulo/manager/compaction/coordinator/CompactionCoordinator.java

```diff
@@ -950,6 +1023,29 @@ public TExternalCompactionList getRunningCompactions(TInfo tinfo, TCredentials c
     return result;
   }
 
+  @Override
+  public TExternalCompactionList getOldestRunningCompactions(TInfo tinfo, TCredentials credentials)
```

Comment:
Knowing this per RG may be useful.  Could return `Map<rgname,topNlist>`.  If one RG is really misbehaving and another RG is slightly misbehaving, then we can not get insight on the slightly misbehaving RG.